src/gui/AutomationPatternView.cpp

```diff
@@ -332,9 +343,11 @@ void AutomationPatternView::paintEvent( QPaintEvent * )
 						MidiTime::ticksPerTact();
 			const float x2 = (float)( width() - TCO_BORDER_WIDTH );
 			if( x1 > ( width() - TCO_BORDER_WIDTH ) ) break;
-
-			p.fillRect( QRectF( x1, 0.0f, x2-x1, it.value() ),
-								lin2grad );
+			if ( gradient() ) {
```

Comment:
Although I quite prefer `if () {` to `if () <newline> {`, we should try to be consistent with the styling of the current file.